x-pack/plugins/lens/public/xy_visualization/to_expression.ts

```diff
@@ -191,16 +191,61 @@ export const buildExpression = (
     return null;
   }
 
+  const isLeftAxis = validDataLayers.some(({ yConfig }) =>
+    yConfig?.some((config) => config.axisMode === 'left')
+  );
+  const isRightAxis = validDataLayers.some(({ yConfig }) =>
+    yConfig?.some((config) => config.axisMode === 'right')
+  );
+
+  const yAxisConfigs: AxisConfig[] = [
+    {
+      position: 'left',
```

Comment:
Should use `Position` from Elastic charts here?

Reply:
done